src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs

```diff
@@ -0,0 +1,105 @@
+﻿using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Apply;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SFA.DAS.RoatpAssessor.Web.Transformers
+{
+    public static class ManagementHierarchyTransformer
+    {
+        private static List<string> InputHeadingTitles = new List<string> { "Name", "Job role", "Years in role", "Months in role", "Part of another organisation", "Organisation details" };
+        private static List<string> TransformHeadingTitles = new List<string> { "Full name", "Job role", "Time in role", "Is this person part of any other organisations?", "Enter the names of all these organisations" };
+
+        public static TabularData Transform(TabularData tabularData)
+        {
+            var managementHierarchy = new TabularData
+            {
+                Caption = null,
+                HeadingTitles = new List<string>(TransformHeadingTitles),
+                DataRows = new List<TabularDataRow>()
+            };
+
+            if (tabularData?.DataRows != null && tabularData.DataRows.Any())
+            {
+                managementHierarchy.DataRows = new List<TabularDataRow>(tabularData.DataRows.Select(dr => { return dr.TransformDataRow(); }));
+            }
+
+            return managementHierarchy;
+        }
+
+        private static TabularDataRow TransformDataRow(this TabularDataRow tabularDataRow)
+        {
+            var dataRow = new TabularDataRow
+            {
+                Id = tabularDataRow.Id,
+                Columns = new List<string>()
+            };
+
+            if (tabularDataRow.Columns != null && tabularDataRow.Columns.Any())
+            {
+                dataRow.Columns = tabularDataRow.Columns.TransformColumns();
+            }
+
+            return dataRow;
+        }
+
+        private static List<string> TransformColumns(this List<string> tabularDataRowColumns)
+        {
+            var columns = new List<string>();
+
+            if (tabularDataRowColumns.Count == InputHeadingTitles.Count)
+            {
+                columns = new List<string>
+                {
+                    tabularDataRowColumns[0],
```

Comment:
Can you add some constants in here so it's easy to see at a glance what the array indexes refer to?

Reply:
Sure, have added these